src/commands/nfts/createNft.ts

```diff
@@ -120,70 +118,100 @@ export const createNft = async (
     services.push('nft-sales')
   }
 
+  const providers: string[] = []
+  argv.providers.forEach((_provider: string) => {
+    if (ethers.utils.isAddress(_provider)) providers.push(_provider)
+  })
+  if (config.nvm.neverminedNodeAddress && 
+      !providers.includes(config.nvm.neverminedNodeAddress) && 
+      ethers.utils.isAddress(config.nvm.neverminedNodeAddress))
+    providers.push(config.nvm.neverminedNodeAddress)
+  
+  logger.info(`Adding Node addresses as asset providers: ${JSON.stringify(providers)}`)
 
   logger.info(`Attaching services to the asset: ${JSON.stringify(services)}`)
 
   logger.info(chalk.dim('\nCreating Asset ...'))
 
   let ddo
 
-  if (argv.nftType === '721') {
-    const nft: Nft721 = await nvm.contracts.loadNft721(argv.nftAddress)
+  let publishMetadata = PublishMetadata.OnlyMetadataAPI
+  if (argv.publishMetadata.toLowerCase() === 'ipfs')
+    publishMetadata = PublishMetadata.IPFS  
+
+  const assetAttributes = AssetAttributes.getInstance({
+      metadata: ddoMetadata,
+      price: assetPrice,
+      serviceTypes: services
+  })
+  let nftAttributes: NFTAttributes
+  
+  if (argv.nftType == 721) {
```

Comment:
```suggestion
  if (argv.nftType === 721) {
```
If we don't want to take both 721 and '721' as true better use `===` instead of `==`